concent_api/utils/helpers.py

```diff
@@ -159,3 +159,10 @@ def calculate_maximum_download_time(size: int) -> int:
             settings.MINIMUM_UPLOAD_RATE,
         ).total_seconds()
     )
+
+
+def join_messages(*msg):
+    list_of_messages = []
+    for msg_part in msg:
```

Comment:
`msg_part` -> `messsage`

Reply:
Like previous comment, `message` would be confusing with golem-messages "message".